packit_service/worker/handlers/github_handlers.py

```diff
@@ -144,7 +144,7 @@ def run(self) -> TaskResults:
 
 
 @configured_as(job_type=JobType.propose_downstream)
-@run_for_comment(command="propose-update")
+@run_for_comment(command="propose-downstream")
```

Comment:
We can have both if we really want some middle step.

Reply:
oh, nice, that's what I've been looking for! thanks